.changeset/violet-needles-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@feature-sliced/steiger-plugin': patch
+---
+
+Bump up filesystem to v3.0.1
```

Comment:
issue: this changelog entry doesn't quite explain what consequences this brings to this package

suggestion: let's specify that this change is done to support `sliceA/@x/sliceB/index.ts` as cross-import public APIs